Content.Server/Botany/SeedPrototype.cs

```diff
@@ -68,21 +69,40 @@ public struct SeedChemQuantity
 public class SeedData
 {
     #region Tracking
+    private string _name = String.Empty;
+    private string _noun = String.Empty;
```

Comment:
make the default value here the same as the field you're replacing

Reply:
```c#
private string _noun = "seeds-noun-seeds";
```
it does not resolves by locale manager so in game it just string
![image](https://user-images.githubusercontent.com/14136326/192070649-ce66ac57-0a99-4076-a145-57462bc48a2b.png)

Reply:
what i meant was having the fallback string (originally just 'seeds') in case somebody leaves out the noun part of the prototype, but even then, it's still ok to have 'You plant the {plant name}', i suppose

Reply:
Better a empty string than an unlocalized one.